src/testUtils/mocks/dataSources.ts

```diff
@@ -657,7 +683,10 @@ export function createTxStub<
  * @param query - apollo document node
  * @param returnValue
  */
-export function createApolloQueryStub(query: DocumentNode, returnData: any): MockApolloClient {
+export function createApolloQueryStub(
+  query: DocumentNode,
+  returnData: any
+): ApolloClient<NormalizedCacheObject> {
   const instance = mockInstanceContainer.apolloInstance;
 
   instance.query = sinon
```

Comment:
This is not doing what you think it does. Sinon has obscure behavior when chaining calls like this (more info: https://nikas.praninskas.com/javascript/2015/07/28/quickie-sinon-withargs-not-working/)

The proper way of doing it is to assign sinon.stub() to a separate variable, call .withArgs and .resolves on that variable, and then assign the variable to instance.query. This will break your tests because you're expecting the .query stub to receive a DocumentNode object, which is not what is actually being passed to it in the code.

Also, createApolloQueryStub should return the stub, not the instance

Finally, `cleanup` in this file should also reset the apollo client instance